docs/userguide/datetime/universaltimescale.md

```diff
@@ -123,31 +123,31 @@ information about a particular timescale. In all of these routines, the
-| Java                   | UDTS_JAVA_TIME              | JAVA_TIME              |
-| Unix                   | UDTS_UNIX_TIME              | UNIX_TIME              |
-| ICU4C                  | UDTS_ICU4C_TIME             | ICU4C_TIME             |
-| Windows FILETIME       | UDTS_WINDOWS_FILE_TIME      | WINDOWS_FILE_TIME      |
-| .NET DateTime          | UDTS_DOTNET_DATE_TIME       | DOTNET_DATE_TIME       |
-| Macintosh (old)        | UDTS_MAC_OLD_TIME           | MAC_OLD_TIME           |
-| Macintosh              | UDTS_MAC_TIME               | MAC_TIME               |
-| Excel                  | UDTS_EXCEL_TIME             | EXCEL_TIME             |
-| DB2                    | UDTS_DB2_TIME               | DB2_TIME               |
-| Unix with microseconds | UDTS_UNIX_MICROSECONDS_TIME | UNIX_MICROSECONDS_TIME |
+| Source                 | ICU4C                         | ICU4J                    |
+| ---------------------- | ----------------------------- | ------------------------ |
+| Java                   | `UDTS_JAVA_TIME`              | `JAVA_TIME`              |
+| Unix                   | `UDTS_UNIX_TIME`              | `UNIX_TIME`              |
+| ICU4C                  | `UDTS_ICU4C_TIME`             | `ICU4C_TIME`             |
+| Windows FILETIME       | `UDTS_WINDOWS_FILE_TIME`      | `WINDOWS_FILE_TIME`      |
+| .NET DateTime          | `UDTS_DOTNET_DATE_TIME`       | `DOTNET_DATE_TIME`       |
+| Macintosh (old)        | `UDTS_MAC_OLD_TIME`           | `MAC_OLD_TIME`           |
+| Macintosh              | `UDTS_MAC_TIME`               | `MAC_TIME`               |
+| Excel                  | `UDTS_EXCEL_TIME`             | `EXCEL_TIME`             |
+| DB2                    | `UDTS_DB2_TIME`               | `DB2_TIME`               |
+| Unix with microseconds | `UDTS_UNIX_MICROSECONDS_TIME` | `UNIX_MICROSECONDS_TIME` |
 
 The routine that gets a particular piece of information about a timescale takes
 an integer constant that identifies the particular piece of information,
 according to the following table:
 
-| Value                | ICU4C                   | ICU4J              |
-| -------------------- | ----------------------- | ------------------ |
-| Precision            | UTSV_UNITS_VALUE        | UNITS_VALUE        |
-| Epoch offet          | UTSV_EPOCH_OFFSET_VALUE | EPOCH_OFFSET_VALUE |
-| Minimum "from" value | UTSV_FROM_MIN_VALUE     | FROM_MIN_VALUE     |
-| Maximum "from" value | UTSV_FROM_MAX_VALUE     | FROM_MAX_VALUE     |
-| Minimum "to" value   | UTSV_TO_MIN_VALUE       | TO_MIN_VALUE       |
-| Maximum "to" value   | UTSV_TO_MAX_VALUE       | TO_MAX_VALUE       |
+| Value                | ICU4C                      | ICU4J                |
+| -------------------- | -------------------------- | -------------------- |
+| Precision            | `UTSV_UNITS_VALUE`         | `UNITS_VALUE`        |
+| Epoch offset          | `UTSV_EPOCH_OFFSET_VALUE` | `EPOCH_OFFSET_VALUE` |
```

Comment:
nit: the spacing for the second column is slightly off. ;)

Reply:
Done.